docs/markdown/Comparisons.md

```diff
@@ -14,7 +14,7 @@ Excellent support for legacy Unix platforms, large selection of existing modules
 
 ### Cons ###
 
-Needlessly slow, complicated, hard to use correctly, unreliable, painful to debug, ununderstandable for most people, poor support for non-Unix platforms (especially Windows).
```

Comment:
It's not a word in `aspell` (though I had to ignore a lot of "errors" from it anyway). Online, I can't find it in Oxford or Cambridge, but can in Merriam-Webster. Maybe it works in talks where you can emphasize is properly, but it just looks odd in text.